nexus/tests/integration_tests/schema.rs

```diff
@@ -589,6 +630,11 @@ impl InformationSchema {
         // the columns diff especially needs this: it can be 20k lines otherwise
         similar_asserts::assert_eq!(self.tables, other.tables);
         similar_asserts::assert_eq!(self.columns, other.columns);
+        similar_asserts::assert_eq!(
+            self.enums,
+            other.enums,
+            "Enums did not match. Members must have the same order in dbinit.sql and migrations. If a migration adds a member, it should be added at the end of the definition in dbinit.sql."
```

Comment:
Nit: Let's update this with Justin's advice to use the AFTER keyword

Reply:
We can specify `BEFORE` or `AFTER`

Reply:
403e1940ca57dda40c8b8684c15751ec869227a4